src/UI/Internal/Tables/FiltersView.elm

```diff
@@ -297,12 +297,13 @@ dialog renderConfig config filter clearMsg applyMsg content =
     in
     overlay config.discardMsg <|
         Element.column
-            [ Element.width fill
-            , zIndex 9
-            , Element.alignTop
-            , Palette.mainBackground
-            , Primitives.defaultRoundedBorders
-            ]
+            (List.append dialogWidthAttrs
```

Comment:
is this only used here? if so it might make sense to define it here as well. Also, I think `(++)` is more idiomatic, although in this case it probably makes more sense to use `(::)` with the two separate elements

Reply:
made this change